op-service/txmgr/txmgr.go

```diff
@@ -608,7 +701,11 @@ func (m *SimpleTxManager) suggestGasPriceCaps(ctx context.Context) (*big.Int, *b
 		basefee = new(big.Int).Set(m.cfg.MinBasefee)
 	}
 
-	return tip, basefee, nil
+	var blobFee *big.Int
+	if head.ExcessBlobGas != nil {
```

Comment:
In a non-urgent follow-up, we should add metrics recording of the field `head.ExcessBlobGas`. Plus, also when we're waiting for confirmations, where we also record fee metrics.
Will create an issue later.

Reply:
done https://github.com/ethereum-optimism/optimism/issues/8851